### ISSUE TITLE
The pad is vibrating before getting the mask of truth

### ISSUE THREAD

**Issue by MO35AB** [Mr-Wiseguy/Zelda64Recomp#250]
**To Reproduce**
1. Go to the entrance to the southern woods, in the tree passage, there is a hole there, the pad will vibrate even if you just started the game.
2. etc.

**Expected behavior**
the game is not supposed to give you hint about secrets until you get the mask

**Desktop (please complete the following information):**
 - OS: [Windows 11]
 - Version: [e.g. 1.0.1]
 - CPU: [intel i7 11th gen]
 - GPU: [NVIDIA RTX 3060]

**Comment by Reonu** [comment]
Hello, I'm afraid you're mistaken. This is how the original game works. 

You might be getting it mixed up with the 3DS behavior. In the 3DS version, since there is no rumble, they made it work with a visual indicator that only activates if you have the mask on truth. In the N64 version, the mask of truth isn't required for this feature at all. The controller will simply rumble when you're near a secret, regardless of masks.

Even though I was 100% sure to begin with, I verified it on my N64 just in case on a savefile just after turning into human Link. The controller vibrates on the spot you mentioned.

Closing since there is no issue here.
